src/servables/tensorrt/BUILD

```diff
@@ -102,10 +104,9 @@ cc_library(
         ":plan_bundle",
         ":plan_bundle_proto",
         "//src/core:constants",
-        "//src/core:logging",
         "//src/core:model_config",
```

Comment:
Since `//src/core:server_header` contains `model_config.h`, can we also remove this?

Reply:
Yes, we probably could. I likely have some redundant dependencies here and there in BUILD files. Feel free to clean them up if you come across them in an are you are working on.